tests/test_optics.py

```diff
@@ -854,7 +838,7 @@ def test_ne():
             galsim.OpticalPSF(lam_over_diam=1.0, defocus=0.1, gsparams=gsp1),
             galsim.OpticalPSF(lam_over_diam=1.0, aberrations=[0, 0, 0, 0, 0.2], gsparams=gsp1),
             galsim.OpticalPSF(lam_over_diam=1.0, nstruts=2, gsparams=gsp1),
-            galsim.OpticalPSF(lam_over_diam=1.0, nstruts=2, strut_thick=0.1, gsparams=gsp1),
+            galsim.OpticalPSF(lam_over_diam=1.0, nstruts=2, strut_thick=0.3, gsparams=gsp1),
```

Comment:
It's because the new `Aperture.__eq__` works by comparing the `.illuminated` image, and not the actual keywords used to generate `.illuminated`.  It happened to be the case here that `strut_thick=0.1` and `strut_thick=0.05` (the default value) produced the same `.illuminated` array, so the Apertures were functionally equal, even though they were generated with slightly different parameters.